Makefile

```diff
@@ -5,8 +5,8 @@ all: docs test
 install: FORCE
 	pip install -e .[dev,profile]
 
-uninstall: FORCE
-	pip uninstall pyro-ppl
+reinstall: FORCE
```

Comment:
When do you need this? If you do an editable install via -e, it should pick up any local changes.

Reply:
whoa, I have uninstalled and installed again and again to test examples, so I made this command. Thx @neerajprad!